tests/func/test_update.py

```diff
@@ -44,6 +44,8 @@ def test_update_import(tmp_dir, dvc, erepo_dir, cached):
 
     assert old_rev != new_rev
 
+    logger = logging.getLogger(__name__)
+    logger.error("#### FUN BEGINS #####")
```

Comment:
If you are only having troubles with this test (and maybe the other one about SHELL), feel free to just mark them as optional and create an issue for it to look right after GHA migration. The main point of this PR is to migrate to GHA, so no reason to hold it just because of those few tests, as we can always just fix them right after.